pkgs/development/python-modules/block-io/default.nix

```
@@ -9,6 +9,13 @@ buildPythonPackage rec {
     sha256 = "4909d58b32ab7f93d3cd83fa4bbe4edef42ab7566f016bdb6a405a0d8b1907c9";
   };
 
+  patches = [
+    (fetchpatch {
+      url = "https://patch-diff.githubusercontent.com/raw/BlockIo/block_io-python/pull/18.patch";
```

```suggestion
      url = "https://github.com/BlockIo/block_io-python/commit/8ec0a0cd1afcfcb439d6dc306d8497dd9e4a0993.patch";
```
The content behind a PR can change.

---

You are right, fixed